src/Sylius/Bundle/ShopBundle/EventListener/NonChannelLocaleListener.php

```diff
@@ -64,7 +64,7 @@ public function restrictRequestLocale(RequestEvent $event): void
         }
 
         $request = $event->getRequest();
-        if (in_array($request->attributes->get('_route'), ['_wdt', '_profiler'])) {
+        if (in_array($request->attributes->get('_route'), ['_wdt', '_profiler', '_profiler_search', '_profiler_search_results'])) {
```

Comment:
However, maybe we can use regexp, to match anything that starts with `_wdt` & `_profiler`?